compatibility_lib/compatibility_lib/configs.py

```diff
@@ -12,7 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Common configs for compatibility_lib."""
+"""Common configs for compatibility_lib and compatibility_server.
+
+Note that a unit test exists for checking that the configs.py file in
+compatibility_lib is the same as the configs.py file in compatibility_server.
+
+The reason for this set up is that these modules need to be isolated from
+each other, but there also needs to be consistency in the objects and data
+in this file since they exist in the same workflow.
+"""
+
+
+import enum
+
+
+class PackageStatus(enum.Enum):
+    """Represents a package's compatibility status.
+
+    The status is based on the results of running 'pip install' and
+    'pip check' on the compatibility server
+
+    UNKNOWN_PACKAGE: package not in whitelist
+    INTERNAL_ERROR: unexpected internal error
+    MISSING_DATA: missing package data from package store
+    SELF_INCOMPATIBLE: pip error when installing self
+    INCOMPATIBLE: pip error when installed with another package
+    OUTDATED_DEPENDENCY: package has an outdated dependency
+    SUCCESS: No issues
+    """
+    UNKNOWN_PACKAGE = 'UNKNOWN_PACKAGE'
+    INTERNAL_ERROR = 'INTERNAL_ERROR'
+    MISSING_DATA = 'MISSING_DATA'
+    SELF_INCOMPATIBLE = 'SELF_INCOMPATIBLE'
+    INCOMPATIBLE = 'INCOMPATIBLE'
+    OUTDATED_DEPENDENCY = 'OUTDATED_DEPENDENCY'
+    SUCCESS = 'SUCCESS'
+
+
+PACKAGE_STATUS_TO_COLOR = {
```

Comment:
Aren't colors a front-end concern that have nothing to do with this library?

Reply:
Oh good point. I committed this change based on your [previous comment](https://github.com/GoogleCloudPlatform/cloud-opensource-python/pull/273#discussion_r266176591) but I should have caught that too.

I'll move this dict somewhere else.